### PR TITLE
Updates /code/game/objects/item/{melee,tcg}/* to follow the style guide

### PR DESCRIPTION
**Pull request by SuperNovaa41**
## About The Pull Request

(did two since they were relatively small)
if you want me to start doing larger chunks at a time lmk, just figured it would be better to do small chunks to prevent merge conflicts and loads of reviewing for one PR

updates every file in the mentioned folders to follow the style guide
just tell me when to stop and ill stop giving you all this work

**regex used for finding errors**

`,[^ \\]`  # For finding non spaced commas
`[^ +/*-=]=[^ =]` # finding non spaced equal signs
`/[A-Za-z]([, )]|$)` # finding single letter variable assignments

## Why It's Good For The **CODEBASE**

i said so

### CODE REVIEW

File: code/game/objects/items/melee/baton.dm [hunk 1]
```diff
@@ -278,12 +278,13 @@
 		attack_self(user)
 
 	sleep(3)
```
Comment (Fikou) [inline code/game/objects/items/melee/baton.dm:280]: 0.3 SECONDS?

File: code/game/objects/items/melee/baton.dm [hunk 2]
```diff
@@ -387,18 +388,19 @@
...
-		qdel(I)
-		qdel(src)
+	if(!istype(item, /obj/item/conversion_kit) || !convertible)
+		return
+	var/turf/source_turf = get_turf(src)
+	var/obj/item/melee/baton/baton = new (source_turf)
+	baton.alpha = 20
+	playsound(source_turf, 'sound/items/drill_use.ogg', 80, TRUE, -1)
+	animate(src, alpha = 0, time = 10)
+	animate(baton, alpha = 255, time = 10)
```
Comment (Mothblocks) [inline code/game/objects/items/melee/baton.dm:401]:
```suggestion
	animate(src, alpha = 0, time = 1 SECONDS)
	animate(baton, alpha = 255, time = 1 SECONDS)
```

File: code/game/objects/items/melee/baton.dm [hunk 3]
```diff
@@ -552,8 +554,8 @@
 	if (cell.charge >= cell_hit_cost)
 		var/scramble_time
 		scramble_mode()
-		for(var/loops in 1 to rand(6,12))
-			scramble_time = rand(5,15)/10
+		for(var/loops in 1 to rand(6, 12))
+			scramble_time = rand(5, 15)/10
```
Comment (Mothblocks) [inline code/game/objects/items/melee/baton.dm:558]:
```suggestion
			scramble_time = rand(5, 15) / (1 SECONDS)
```

File: code/game/objects/items/melee/baton.dm [hunk 4]
```diff
@@ -625,23 +626,22 @@
...
-		var/caught = hit_atom.hitby(src, FALSE, FALSE, throwingdatum=throwingdatum)
-		var/mob/thrown_by = thrownby?.resolve()
-		if(isliving(hit_atom) && !iscyborg(hit_atom) && !caught && prob(throw_stun_chance))//if they are a living creature and they didn't catch it
-			finalize_baton_attack(hit_atom, thrown_by, in_attack_chain = FALSE)
-		if(thrown_by && !caught)
-			addtimer(CALLBACK(src, /atom/movable.proc/throw_at, thrown_by, throw_range+2, throw_speed, null, TRUE), 1)
-	else
+	if(!active)
 		return ..()
+	var/caught = hit_atom.hitby(src, FALSE, FALSE, throwingdatum = throwingdatum)
```
Comment (Mothblocks) [inline code/game/objects/items/melee/baton.dm:639]: Can you make the `FALSE, FALSE` named variables?

File: code/game/objects/items/melee/energy.dm [hunk 5]
```diff
@@ -249,19 +249,16 @@
...
-		else
-			hacked = TRUE
-			sword_color_icon = "rainbow"
-			to_chat(user, span_warning("RNBW_ENGAGE"))
-			if(force >= active_force)
-				icon_state = "[initial(icon_state)]_on_rainbow"
-				user.update_inv_hands()
+/obj/item/melee/energy/sword/saber/multitool_act(mob/living/user, obj/item/tool)
+	if(hacked)
+		to_chat(user, span_warning("It's already fabulous!"))
```
Comment (Mothblocks) [inline code/game/objects/items/melee/energy.dm:254]: Early-return here

File: code/game/objects/items/melee/misc.dm [hunk 6]
```diff
@@ -258,13 +258,13 @@
...
+/obj/item/melee/supermatter_sword/proc/consume_turf(turf/turf)
+	var/oldtype = turf.type
+	var/turf/newT = turf.ScrapeAway(flags = CHANGETURF_INHERIT_AIR)
 	if(newT.type == oldtype)
 		return
-	playsound(T, 'sound/effects/supermatter.ogg', 50, TRUE)
-	T.visible_message(span_danger("[T] smacks into [src] and rapidly flashes to ash."),\
+	playsound(turf, 'sound/effects/supermatter.ogg', 50, TRUE)
+	turf.visible_message(span_danger("[turf] smacks into [src] and rapidly flashes to ash."),\
 	span_hear("You hear a loud crack as you are washed with a wave of heat."))
```
Comment (Mothblocks) [inline code/game/objects/items/melee/misc.dm:268]:
```suggestion
	turf.visible_message(
		span_danger("[turf] smacks into [src] and rapidly flashes to ash."),
		span_hear("You hear a loud crack as you are washed with a wave of heat."),
	)
```

File: code/game/objects/items/melee/misc.dm [hunk 7]
```diff
@@ -386,25 +386,26 @@
...
+		to_chat(user, span_notice("You send [held_sausage] towards [target]."))
+		playsound(src, 'sound/items/rped.ogg', 50, TRUE)
+		beam = user.Beam(target, icon_state = "rped_upgrade", time = 10 SECONDS)
+	else if (user.Adjacent(target))
+		to_chat(user, span_notice("You extend [src] towards [target]."))
+		playsound(src.loc, 'sound/weapons/batonextend.ogg', 50, TRUE)
 		finish_roasting(user, target)
+		return
+	else
+		return
```
Comment (Mothblocks) [inline code/game/objects/items/melee/misc.dm:401]:
```suggestion
	else
		return
```

`finish_roasting` will be called regardless

File: code/game/objects/items/tcg/tcg.dm [hunk 8]
```diff
@@ -306,8 +306,8 @@ GLOBAL_LIST_EMPTY(tcgcard_radial_choices)
 	var/list/temp_deck = contents.Copy()
 	contents = reverse_range(temp_deck)
 	//Now flip the cards to their opposite positions.
-	for(var/a in 1 to contents.len)
-		var/obj/item/tcgcard/nu_card = contents[a]
+	for(var/i in 1 to contents.len)
+		var/obj/item/tcgcard/nu_card = contents[i]
```
Comment (Mothblocks) [inline code/game/objects/items/tcg/tcg.dm:310]:
```suggestion
	for(var/obj/item/tcgcard/nu_card as anything in contents)
```

File: code/game/objects/items/tcg/tcg.dm [hunk 9]
```diff
@@ -526,9 +526,9 @@ GLOBAL_LIST_EMPTY(tcgcard_radial_choices)
 				toReturn += "[pack.type] has a rarity [template.rarity] on the card [template.id] that does not exist\n"
 				continue
 		//Lets run a check to see if all the rarities exist that we want to exist exist
-		for(var/I in pack.rarity_table)
-			if(!GLOB.cached_cards[pack.series][I])
-				toReturn += "[pack.type] does not have the required rarity [I]\n"
+		for(var/_rarity in pack.rarity_table)
```
Comment (Mothblocks) [inline code/game/objects/items/tcg/tcg.dm:529]: Why the underscore prefix?

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/game/objects/items/tcg/tcg.dm (hunk 9 above)
Comment (SuperNovaa41) [reply code/game/objects/items/tcg/tcg.dm:528]: rarity is a var on `src` and i couldn't think of anything else to use there that would make sense

Comment (Mothblocks) [reply code/game/objects/items/tcg/tcg.dm:528]: Fair, what about `pack_rarity`?

Just `rarity` *should* also work, IIRC, but I understand not wanting to repeat it.

Comment (SuperNovaa41) [reply code/game/objects/items/tcg/tcg.dm:528]: works for me

**Pull request merged**